scripts/examples/openiotsdk_example.sh

```diff
@@ -42,6 +42,12 @@ FVP_NETWORK="user"
 
 readarray -t TEST_NAMES <"$CHIP_ROOT"/src/test_driver/openiotsdk/unit-tests/testnames.txt
 
+declare -a APP_NAMES
```

Comment:
Nice improvement, I think you can put it in a separate PR then it can be fast merged into development.